client/src/component/MilestoneTable.js

```diff
@@ -11,8 +15,9 @@ const MilestoneTable = () => {
 				</thead>
 				<tbody>
 					<tr>
-						<td>5 out of 7</td>
-						<td>3</td>
+						<td>{githubData}</td>
```

Comment:
the github Data is an object so you reach out the data inside it fro example GithubData.total_count gives you a number of pull request which user has on cyf reposotiry (we know fthey are not all mandatory we can replace this later when we filter data), next pont is we are checking if it is out of 26 here not in codewars column